astra-widgets.php

```diff
@@ -25,7 +25,7 @@
 define( 'ASTRA_WIDGETS_BASE', plugin_basename( ASTRA_WIDGETS_FILE ) );
 define( 'ASTRA_WIDGETS_DIR', plugin_dir_path( ASTRA_WIDGETS_FILE ) );
 define( 'ASTRA_WIDGETS_URI', plugins_url( '/', ASTRA_WIDGETS_FILE ) );
-define( 'ASTRA_WIDGETS_VER', '1.2.5' );
+define( 'ASTRA_WIDGETS_VER', '1.2.6' );
```

Comment:
@shubhw12 we need to update the plugin version as well - `* Version: 1.2.5` Starting of this file.